src/snyk/base/modules/snykLib.ts

```diff
@@ -141,7 +142,9 @@ export default class SnykLib extends BaseSnykModule implements ISnykLib {
 
     try {
       const oldResult = this.ossService.getResult();
-      const result = await this.ossService.test(manual, reportTriggeredEvent);
+      const result: OssResult = await this.ossService.test(manual, reportTriggeredEvent);
+      await this.advisorService?.setScores(result);
```

Comment:
Advisor should not relate to OSS Service at all. OSS service is responsible for running CLI test for Open Source vulnerabilities. Advisor needs only opened manifest file to be able to fetch the score. These are two distinct features and products.

Reply:
So in order to get vulnerabilities for a package I need to use CLIService instead?